src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -181,13 +225,36 @@ namespace PhotovoltaicThermalCollectors {
 
         void calculate(EnergyPlusData &state);
 
+        void SimplePVTcalculate(EnergyPlusData &state);
+
+        void BIPVTcalculate(EnergyPlusData &state);
+
+        void BIPVT_MaxHeatGain_calculate(
+            EnergyPlusData &state, Real64 tsp, std::string Mode, Real64 &bfr, Real64 &q, Real64 &tmixed, Real64 &ThEff, Real64 &tpv);
+
+        void solve_lin_sys_back_sub(Real64 jj[9], Real64 f[3], Real64 (&y)[3]);
```

Comment:
If you look in the wiki and the code, you'll find conflicting guidance on naming conventions (sorry). The current preferred style is camelCase for function names, starting with the active word. So, these would be:
`calculateSimplePVT, calculateBIPVT, calculateBIPVTMaxHeatGain, solveLinSysBackSub`.

Reply:
changed names of methods as suggested in this comment

Reply:
names of methods updated as suggested